### ISSUE TITLE
[WebView] Problem with Modifier.fillMaxHeight() (and Modifier.fillMaxSize())

### ISSUE THREAD

**Issue by alirahimpour89** [google/accompanist#1513]
**Description**
Since you added a frame layout around the webview some websites have started get weird sizing problems.
The problem is visible on websites that isn't that tall and doesn't need scrolling. I get white space below the webview.

**Steps to reproduce**
```
val url = "https://www.google.com"
val state = rememberWebViewState(url = url)
WebView(
   modifier = Modifier.fillMaxSize(),
   state = state,
   onCreated = { it.settings.javaScriptEnabled = true },
)
```

**Expected behavior** 
When using Modifier.fillMaxSize or fillMaxHeight the WebView should be taking up all available space. 

**Additional context**
I have tried adding this to your code:
`parentLayout.layoutParams = ViewGroup.LayoutParams(width, height)`
And it works as expected and looks like it did before you added the FrameLayout.

**Comment by bentrengrove** [comment]
Happy to accept a PR if you want to contribute that. I should be able to get to it soon if not

**Comment by alirahimpour89** [comment]
One of the checks timed out, can I rerun it or do you have to? It's a timeout so I hope it works on next run.

**Comment by bentrengrove** [comment]
Fixed in #1515
